src/test/java/org/assertj/core/api/map/MapAssert_containsOnlyKeys_with_Path_Test.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.api.map;
+
+import static org.mockito.Mockito.verify;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.assertj.core.api.MapAssert;
+import org.assertj.core.api.MapAssertBaseTest;
+import org.junit.jupiter.api.DisplayName;
+
+/**
+ * Tests for <code>{@link MapAssert#containsOnlyKeys(Iterable)}</code> with {@link Path} parameter.
+ *
+ * @author Stefano Cordio
+ */
+@DisplayName("MapAssert containsOnlyKeys(Path)")
+class MapAssert_containsOnlyKeys_with_Path_Test extends MapAssertBaseTest {
+
+  private final Path path = Paths.get("file");
+
+  @Override
+  protected MapAssert<Object, Object> invoke_api_method() {
+    return assertions.containsOnlyKeys(path);
+  }
+
+  @Override
+  protected void verify_internal_effects() {
+    verify(maps).assertContainsOnlyKeys(getInfo(assertions), getActual(assertions), (Object) this.path);
```

Comment:
Minor: remove this or add it in L36
Add a comment explaining why we need to cast to `Object` as it is not super obvious (to me at least)